src/NServiceBus.RavenDB/NServiceBus.RavenDB.csproj

```
@@ -15,7 +15,7 @@
     <PackageReference Include="NServiceBus" Version="[8.0.0-rc.3, 9)" />
     <PackageReference Include="Obsolete.Fody" Version="5.3.0" PrivateAssets="All" />
     <PackageReference Include="Particular.Packaging" Version="2.2.0" PrivateAssets="All" />
-    <PackageReference Include="RavenDB.Client" Version="[5.2.1, 6.0.0)" />
+    <PackageReference Include="RavenDB.Client" Version="[5.2.100, 6.0.0)" />
```

```suggestion
    <PackageReference Include="RavenDB.Client" Version="[5.2.107, 6.0.0)" />
```

Contains the required bug fix and therefore is the required minimum version

> [Session] fixed possible NRE when updating compare exchange metadata in the session